skyportal/tests/api/test_spectrum.py

```diff
@@ -6,6 +6,74 @@
 import datetime
 
 
+def test_token_user_get_range_spectrum(
```

Comment:
test improperly formatted date string 

test that the following ISO date strings for `min_date` / `max_date` all work


2020-12-1
2020-12-18T03:52:47+00:00
2020-12-18T03:52:47Z
20201218T035247Z

Reply:
That last example throws an error for astropy.time.Time with format='isot'.  Tests for the others were implemented.

Reply:
How about 

2020-12-18T03:52:47+07:00

?

Does it handle the time zone correctly?

Reply:
I also added a test for when the parameter instrument_ids is not present.

Reply:
Yes, but you have to use the html code '&plus;' and not the character '+', which generates a space.